src/Data/Config.php

```diff
@@ -196,19 +220,38 @@ public function graphql_wp_query_cursor_pagination_stability( string $orderby, W
 	 * @return string
 	 */
 	public function graphql_wp_query_cursor_pagination_support( string $where, WP_Query $query ) {
-
-		// if it's not a graphql request, return early
+		// Bail early if it's not a GraphQL Request.
 		if ( true !== is_graphql_request() ) {
 			return $where;
 		}
 
-		// apply the after cursor to the query
+		/**
+		 * If pre-filter hooked, return $pre_where.
+		 * 
+		 * @param null|mixed $pre_where The pre-filtered WHERE clause of the query.
```

Comment:
```suggestion
		 * @param null|string $pre_where The pre-filtered WHERE clause of the query.
```

This also needs to be changed on the `@return` tag, and on the other filters.